community/mei-by-laws.md

```diff
@@ -73,41 +77,51 @@ The Board is self-organizing; that is, there will be no formal regulations on ho
+The administrative chair takes responsibility for calling Board meetings. The designated Host Institution Board members act as _liaison_ with the host organizations. Moreover, the chair of the Board is the main representative of the whole MEI Community and thus has representational functions and obligations and may act on behalf of the whole MEI community on such occasions. (N.B. the above is not to say that the chair of the Board has more influence and decision competence than any other member of the Board. The chair of the Board is simply be “first among equals.”)
 
 The two Technical Co-chairs take responsibility for organizing and chairing the Technical Team, see details below (6 Technical Team).
 
-### 5.3 Terms & Elections
+The two Elections Officers will take care of running the yearly MEI elections. The Elections Officers are appointed by the Board from amongst the MEI community and shall not be running for election themselves.
 
-The elected members of the Board each are elected for a three year term. Terms on the Board are staggered so that three positions are open for election each year. Terms of office begin January 1 at 2 pm UTC.
+### 5.2 Meetings
 
-Exception to annual elections and term lengths will be made for the initial election of the Board (see Initial election of the Board) and in case a Board member resigns.
+The Board holds regular, virtual or in-person meetings, e.g. approximately every three months. As schedules and funding permit, the Board meets in person at least once per year. For every meeting, an agenda shall be published online prior to the meeting and minutes of the meeting shall be provided in a timely manner following the meeting. Depending on the availability of funding, the members of the Board may be refunded travel costs for this meeting.
+
+### 5.3 Terms & Elections
 
-Board members may resign from their commitments at any time but are expected to give adequate notice. After accepting the resignation, the Board may decide on how to proceed concerning the newly vacant position. Options should include but are not limited to holding a special election, appointing a replacement, or leaving the position vacant.
+The elected members of the Board each are elected for a three year term. Terms on the Board are staggered so that three positions are open for election each year. Terms of office begin January 1 at 2 pm UTC.
 
-In case of a special election to fill a vacancy between regular elections the newly elected Board member will serve for the remainder of the term of the vacant position.
+Exception to annual elections and term lengths will be made for the initial election of the Board and in case a Board member resigns.
```

Comment:
```suggestion
Exceptions to annual elections and term lengths will be made for the initial election of the Board and in case a Board member resigns.
```